tests/fuzz/src/read_network-fuzzer.cc

```diff
@@ -38,7 +38,9 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
     try {
         InferenceEngine::Core ie;
         InferenceEngine::CNNNetwork network = ie.ReadNetwork(net, weights_blob);
-    } catch (const InferenceEngine::details::InferenceEngineException& error) {
+    } catch (const InferenceEngine::details::InferenceEngineException&) {
+        return 0;  // fail gracefully on expected exceptions
+    } catch (const std::exception&) {
```

Comment:
Do we want to catch all `catch(...)` or only based on `std::exception`?

Reply:
No, we don't want to catch all exceptions. Instead, we want to catch if IE throws an unexpected type of exception